components/Slots/Slots.jsx

```diff
@@ -0,0 +1,55 @@
+import styles from "./Slots.module.css";
+import en from "javascript-time-ago/locale/en";
+import TimeAgo from "javascript-time-ago";
+
+TimeAgo.addDefaultLocale(en);
+
+const timeAgo = new TimeAgo("en-US");
+const DEFAULT_BRANCH_NAME = "main";
+
+export function Slots({ slots = [], url }) {
+  const slotsWithoutBot = slots.filter((s) => s.commitAuthor !== "web-flow");
```

Comment:
Let's  add const:
`const GITHUB_BOT_AUTHOR = "web-flow"`

Reply:
did it in this pr https://github.com/ekaterinburgdev/projects/pull/10
moved to the be